### ISSUE TITLE
Feature Request: Add Gradle Support

### ISSUE THREAD

**Issue by JJTech0130** [Beta8397/virtual_robot#86]
Would it be possible to configure this repo with gradle, such that one monolithic repository can build both the production code and the simulator with different targets? I don’t have time to invest in this, but it seems possible and would be a very nice feature to have.

**Comment by jkenney2** [comment]
Agreed. I've made a couple of attempts with both gradle and maven. I have no experience with those systems, and kept running into errors. Learning about them is on my to-do list. I'll leave this issue open.

**Comment by JJTech0130** [comment]
I'm working on it, but Java modules are a headache. I might just stick to Java 8 for now and see if that solves it...
